.github/workflows/ci_pullrequest.yml

```diff
@@ -487,3 +435,52 @@ jobs:
         run: |
           git fetch --no-tags origin +refs/heads/${{ github.base_ref }}:refs/remotes/origin/${{ github.base_ref }}
           poetry run towncrier check --compare-with origin/${{ github.base_ref }}
+  container-build-test:
+    name: "Test Container Build (amd64 only on Python 3.11)"
+    runs-on: "ubuntu-22.04"
```

Comment:
Any reason why this is the only test that runs on ubuntu-22.04 instead of ubuntu-20.04?

Reply:
I think @snaselj updated this to use the new version when he updated the container build process. We're probably ok to update the others but we need to account for the time it will take to research and test in a separate issue.